packages/expo-secure-store/test/mocking.js

```diff
@@ -0,0 +1,63 @@
+import { Platform } from 'react-native';
```

Comment:
can we move these functions to `jest-expo`? 🤔 that sounds reasonable as it's going to be used in several packages.

Reply:
Created an issue for this (https://github.com/expo/expo/issues/3319)